cmd/zetacored/parse_genesis.go

```diff
@@ -172,6 +177,18 @@ func ImportDataIntoFile(
 	return nil
 }
 
+func ModifyStakingState(
```

Comment:
Can we add a comment on the purpose of this change? Is it related to the PR?

Reply:
reverted , the changes to parse_genesis.go were pushed in by mistake